recipes/libde265/run_test.sh

```diff
@@ -0,0 +1,5 @@
+git clone https://github.com/strukturag/libde265-data.git
+dec265 -q -c -f 100 ./libde265-data/IDR-only/paris-352x288-intra.bin
+dec265 -t 4 -q -c -f 100 ./libde265-data/IDR-only/paris-352x288-intra.bin
+dec265 -0 -q -c -f 100 ./libde265-data/RandomAccess/paris-ra-wpp.bin
+dec265 -0 -t 4 -q -c -f 100 ./libde265-data/RandomAccess/paris-ra-wpp.bin
```

Comment:
These tests are mainly to ensure that the libraries were packages correctly. Do your binaries have a `--help` or `--version` option that will just print and return 0? If so, please do that for all the binaries: encode, decode, and hdrcopy. Full unit tests are not really necessary.